exchanges/gateio/gateio_websocket.go

```diff
@@ -339,13 +340,25 @@ func (g *Gateio) WsHandleData() {
 	}
 }
 
+// GenerateAuthenticatedSubscriptions Adds authenticated subscriptions to websocket to be handled by ManageSubscriptions()
+func (g *Gateio) GenerateAuthenticatedSubscriptions() {
+	var channels = []string{"balance.subscribe", "order.subscribe"}
+	subscriptions := []exchange.WebsocketChannelSubscription{}
```

Comment:
This might bring up an issue with some linters as its a non nil nil slice.
```go
var subscriptions []exchange.WebsocketChannelSubscription
```
instead